search-service/src/main/kotlin/com/egm/stellio/search/model/AttributeInstanceResult.kt

```diff
@@ -0,0 +1,12 @@
+package com.egm.stellio.search.model
+
+import java.net.URI
+import java.time.ZonedDateTime
+
+data class AttributeInstanceResult(
+    val attributeName: String,
+    val instanceId: URI? = null,
+    val datasetId: URI? = null,
+    val value: Any? = null,
```

Comment:
value should not be null ... but maybe we have some legacy? in which case, we should delete records with a null vale.

Reply:
From this test `it should gracefully handle null values in the history of a non-numeric temporal attribute` in TemporalEntityAttributeServiceTests, I understand that the value is nullable

Reply:
Hmm, yes, I remember adding that because the SigFox IoT Agent pushed null values (do you confirm @vraybaud?).

Reply:
Cf https://github.com/stellio-hub/stellio-context-broker/commit/b3cf593fc8a7026a7cca1b54254dd5e51a1e57f3

Reply:
So, what I suggest:
- Only create an AttributeInstance if it has a non null value or measuredValue
- Prepare a migration script to remove from DB the existing records that have no value and no measuredValue
- Then we can be sure an AttributeInstanceResult has a value

Does it sound good?

Reply:
Indeed null values were pushed because textual values were not handled back then.

Reply:
> Does it sound good?
Yes